docs/build-on-polkadot/xcm/asset-transfer-api.md

```diff
@@ -0,0 +1,491 @@
+
+- Execute asset transfers to other parachains or locally within the same chain
+- Facilitate transactions involving system parachains like Asset Hub (Polkadot and Kusama)
+
+The Asset Transfer API supports Parachain to Parachain transfers, currently limited to XCM V2, except for primary asset transactions. 
+
+By using this API, developers can manage asset transfers more efficiently, reducing the complexity associated with cross-chain transactions and enabling smoother operations within the ecosystem.
+
+For additional support and information, please reach out through [GitHub Issues](https://github.com/paritytech/asset-transfer-api/issues){target=_blank}.
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- [Node.js](https://nodejs.org/en/){target=_blank} (recommended version 21 or greater)
+- Package manager - [npm](https://www.npmjs.com/){target=_blank} should be installed with Node.js by default. Alternatively, you can use other package managers like [Yarn](https://yarnpkg.com/){target=_blank}
+
+## Install Asset Transfer API
+
+To use `asset-transfer-api`, you need a JavaScript project. If you don't have one, you can create a new one:
+
+1. Create a new directory for your project:
+    ```bash
+    mkdir my-asset-transfer-project \
+    && cd my-asset-transfer-project
+    ```
+
+2. Initialize a new project:
+    ```bash
+    npm init -y
+    ```
+
+Once you have a project set up, you can install the asset-transfer-api package:
+
+```bash
+npm install @substrate/asset-transfer-api@{{build_on_polkadot.xcm.asset_transfer_api.version}}
+```
+
+!!!note
+    This documentation covers version `{{build_on_polkadot.xcm.asset_transfer_api.version}}` of Asset Transfer API.
+
+## Set Up Asset Transfer API
+
+To initialize the Asset Transfer API, you need three key components:
+
+- A Polkadot.js API instance
+- The `specName` of the chain
+- The XCM version to use
+
+<!-- You can set up the API in two ways. Choose the option that best fits your project's structure and requirements. Both methods will result in a fully initialized AssetTransferApi instance ready for use:
+
+### Using an Existing Polkadot.js API
+
+If you already have an initialized Polkadot.js API instance, you can use it to set up the Asset Transfer API:
+
+```javascript
+// Assuming 'api' is your initialized Polkadot.js API instance
+const { specName } = await api.rpc.state.getRuntimeVersion();
+const safeXcmVersion = await fetchSafeXcmVersion(api);
+
+const assetsApi = new AssetTransferApi(api, specName, safeXcmVersion);
+```
+-->
+
+### Using helper function from library
+
+For a simpler setup process, you can leverage the `constructApiPromise` helper function provided by the library. 
```

Comment:
I'd like to add a concise sentence that provides further information about this helper, like explaining what it does actually or why we take advantage of it